css/main.css

```diff
@@ -17,16 +17,23 @@
     --color-sec: #0e0822;
     --text-pri: #9a86cb;
     --text-sec:#272d37;
-    --bg-color-pri: #f3f4f6;
-    --bg-color-sec: #d1d5db;
+    --color-white: #f3f4f6;
+    --color-off-white: #d1d5db;
+    --color-green:hsl(120, 100%, 25%);
+    --color-off-green:hsl(151, 41%, 79%);
+    --color-yellow:hsl(35, 100%, 50%);
+    --color-yellow:hsl(60, 100%, 25%);
+    --color-off-yellow:hsl(60, 41%, 79%);
+    --color-red:hsl(0, 100%, 25%);
+    --color-off-red:hsl(0, 41%, 79%);
 }
 
 html{
```

Comment:
Can be moved to reset.css if you have one.

Reply:
Done